cts/cli/regression.tools.exp

```diff
@@ -2773,9 +2785,9 @@ Set 'test-primitive' option: id=test-primitive-meta_attributes-is-managed name=i
 =#=#=#= End test: Update existing resource meta attribute - OK (0) =#=#=#=
 * Passed: crm_resource   - Update existing resource meta attribute
 =#=#=#= Begin test: Create a resource meta attribute in the parent =#=#=#=
-Set 'test-clone' option: id=test-clone-meta_attributes-is-managed set=test-clone-meta_attributes name=is-managed value=true
+Set 'test-clone' option: id=test-clone-meta_attributes-is-managed name=is-managed value=true
```

Comment:
Do you know why the set type disappeared?

Reply:
It's back again)